<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code editor viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="insertAction"/>
   <addaction name="removeAction"/>
   <addaction name="separator"/>
   <addaction name="importAction"/>
   <addaction name="separator"/>
   <addaction name="clearAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="importAction">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icons/icons/import.svg</normaloff>:/icons/icons/import.svg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import .xml from folder</string>
   </property>
  </action>
  <action name="insertAction">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icons/icons/add.svg</normaloff>:/icons/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add new entry</string>
   </property>
  </action>
  <action name="removeAction">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icons/icons/remove.svg</normaloff>:/icons/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected entry</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/icons/icons/clear.svg</normaloff>:/icons/icons/clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear all entries</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
